misc/rapiddisk-rootdev/ubuntu/rapiddisk_sub.orig

```diff
@@ -1,9 +1,13 @@
 #!/bin/sh
 
+. /scripts/functions
+
+modprobe -v dm-writecache
 modprobe -v rapiddisk
 modprobe -v rapiddisk-cache
 
+rapiddisk -a 1
```

Comment:
Can you please explain why we are created this small volume and deleting it later?

Reply:
Sure, under Ubuntu 20.04.3 I got a weird error on boot:

```
rapiddisk 7.2.1
Copyright 2011 - 2021 Petros Koutoupis

rd0 is currently mounted. Please "umount" and retry.
```

But if I create a first ramdisk (rd0), an then a second one (rd1), the latter attaches without problem to the boot device. I know this is just a workaround: do you have any clue?

Reply:
Can you add a comment in-line explaining this please?

Reply:
> Can you add a comment in-line explaining this please?

Do you mean a comment in the script or in the generated initrd files? No problem anyway.

BTW, under other installations of Ubuntu 20.04.3 (an Amazon AWS one for example) the workaround is not needed - it works fine creating a single `rd0` ramdisk an attaching it.

On other Ubuntu machines, I noticed that `rapiddisk` loads at boot in `wa` and `wt` mode, while in `wb` mode exits with an error - I'll try to gather more debug infos about this behaviour.

Reply:
I meant in line with the code to explain why it is being done in the first place. I would hate to revisit this in the future or have someone else look at it and ask the same questions. Thanks you.